src/filterregistry.c

```diff
@@ -82,6 +88,9 @@ int (*snoopy_filterregistry_ptrs []) (char *logMessage, char const * const arg)
 #ifdef SNOOPY_CONF_FILTER_ENABLED_only_uid
     snoopy_filter_only_uid,
 #endif
+#ifdef SNOOPY_CONF_FILTER_ENABLED_only_uid
```

Comment:
oops--copypasta error here.  Should be `#ifdef SNOOPY_CONF_FILTER_ENABLED_only_tty`. I'll fix, and might as well rearrange so the ifdefs are in alphabetical order.